package.json

```diff
@@ -45,17 +45,17 @@
   },
   "dependencies": {
     "vl-mapactions": "^2.1.0",
-    "vl-ui-select": "^4.0.0",
-    "vl-ui-core": "^5.0.1"
+    "vl-ui-core": "^5.1.0",
+    "vl-ui-select": "^4.0.1"
   },
   "peerDependencies": {
     "vl-mapactions": "^2.1.0",
-    "vl-ui-select": "^4.0.0",
-    "vl-ui-core": "^5.0.1"
+    "vl-ui-core": "^5.1.0",
+    "vl-ui-select": "^4.0.1"
   },
   "devDependencies": {
-    "vl-ui-button": "^4.0.0",
-    "vl-ui-icon": "^4.0.0",
+    "vl-ui-button": "^5.0.1",
+    "vl-ui-icon": "^5.0.0",
```

Comment:
Is deze dependency eigenlijk nog wel nodig? Ik zie geen verwijzingen naar icon in de code.

Reply:
Nee blijkbaar niet, ik heb het weggehaald